cdap-proto/src/test/java/co/cask/cdap/proto/audit/AuditMessageTest.java

```diff
@@ -62,11 +62,12 @@ public void testAccessMessage() throws Exception {
     String flowAccessJson =
       "{\"version\":1,\"time\":2000,\"entityId\":{\"namespace\":\"ns1\",\"stream\":\"stream1\"," +
         "\"entity\":\"STREAM\"},\"user\":\"user1\",\"type\":\"ACCESS\",\"payload\":{\"accessType\":\"WRITE\"," +
-        "\"accessor\":{\"namespace\":\"ns1\",\"application\":\"app1\",\"type\":\"Flow\",\"program\":\"flow1\"," +
-        "\"run\":\"run1\",\"entity\":\"PROGRAM_RUN\"}}}";
+        "\"accessor\":{\"namespace\":\"ns1\",\"application\":\"app1\",\"version\":\"v1\",\"type\":\"Flow\"," +
+        "\"program\":\"flow1\",\"run\":\"run1\",\"entity\":\"PROGRAM_RUN\"}}}";
     AuditMessage flowAccess =
       new AuditMessage(2000L, Ids.namespace("ns1").stream("stream1"), "user1", AuditType.ACCESS,
-                       new AccessPayload(AccessType.WRITE, Ids.namespace("ns1").app("app1").flow("flow1").run("run1")));
+                       new AccessPayload(AccessType.WRITE, Ids.namespace("ns1").app("app1", "v1").flow("flow1")
```

Comment:
nit: unrelated to this PR, but can you replace ``Ids.namespace`` with ``new NamespaceId``?